ui/src/components/gameContainer/Goban.vue

```diff
@@ -48,6 +48,15 @@
                 src="0.png"
               />
             </div>
+            <div class="tileAlpha" v-else-if="tile === 0 && invalidMoves.includes((posX + (posY * size)))">
+              <img
+                v-on:mouseover="mouseOverError(posX + (posY * size), currentPlayer)"
+                v-on:mouseleave="mouseOut(posX + (posY * size), tile)"
+                :id="posX + (posY * size)"
+                class="tile0"
```

Comment:
I'm guessing this is the case where red tile should apprear, and so you wanted to use `error.png`?

Reply:
Nope should only be error.png if you mouse over it, for me that's a better interface so you don't have the harsh red tiles always visible

Reply:
Agree ;)